inference-engine/src/multi_device/multi_device_plugin.cpp

```diff
@@ -289,6 +289,7 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
                                                                metaDevices,
                                                                multiNetworkConfig,
                                                                enablePerfCounters);
+    SetExeNetworkInfo(impl, network.getFunction());
```

Comment:
later we override this graph by one of the device graphs, do we need it?

Reply:
removed